### PR TITLE
Update general.rst

### PR DESCRIPTION
**Pull request by ismailbayram**

### CODE REVIEW

File: docs/general.rst [hunk 1]
```diff
@@ -47,14 +47,14 @@ Get the exchange info for a particular symbol
 
     info = client.get_symbol_info('BNBBTC')
 
-`Get All Coins Info <binance.html#binance.client.Client.get_all_coins_info>`_
+`Get All Coins Info <binance.html#binance.client.Client.get_all_tickers>`_
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```
Comment (AlexNik) [inline docs/general.rst:51]: For pretty docs you should copy `^` to the last symbol of previous line.

**Review by AlexNik — CHANGES_REQUESTED** [review]

File: docs/general.rst (hunk 1 above)
Comment (ismailbayram) [reply docs/general.rst:51]: I dont understand, I've just changed the name of method.

Comment (AlexNik) [reply docs/general.rst:51]: I see.
But the line 51 is bad. It had a mistake before and now also has.
You should add `^` symbol that the line 50 and 51 have the same lenght.
Eg. like in 59 and 60.

Comment (Kyzegs) [reply docs/general.rst:51]: Basically, just change this, @ismailbayram:
```python
`Get All Coins Info <binance.html#binance.client.Client.get_all_tickers>`_
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```
To this:
```python
`Get All Coins Info <binance.html#binance.client.Client.get_all_tickers>`_
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

Correct me if I'm wrong.

**Comment by sammchardy** [comment]
thanks @ismailbayram

**Pull request merged**